account_journal_lock_date/__manifest__.py

```diff
@@ -1,17 +1,16 @@
-# -*- coding: utf-8 -*-
 # Copyright 2017 ACSONE SA/NV
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     'name': 'Account Journal Lock Date',
     'summary': """
         Lock each journal independently""",
-    'version': '10.0.1.0.0',
+    'version': '12.0.1.0.0',
     'license': 'AGPL-3',
     'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
-    'website': 'https://acsone.eu/',
+    'website': 'https://github.com/OCA/account-financial-tools',
     'depends': [
-        'account_permanent_lock_move',
+        'account',
```

Comment:
Not corresponding to DESCRIPTION.rst, why has this dependency changed ?

Reply:
That's because the previous module name was not correct. A better one is `account_journal_permanent_lock_date`, and I suppose that one will be added later.